Makefile

```diff
@@ -135,8 +136,9 @@ audit: install-audit audit-CI
 install-audit:
 	cargo install --force cargo-audit
 
+# TODO - remove when updated in LH(?)
 audit-CI:
-	cargo audit
+	cargo audit --ignore RUSTSEC-2024-0399
```

Comment:
Are we failing this RUSTSEC?

Maybe we address this first in a separate PR?

Reply:
Will do.
Removed.